functional/basic-attestation-on-localhost/payload-script/sudo_script.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo -v
```

Comment:
We need to run some command in order to gain (and verify) root priviledges, e.g. `id`.
Later, we should check the script output to verify it has been executed as `root`.